src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -42,8 +42,11 @@
 	private final static Noun m_minute = new Noun("time.minute");
 	private final static Noun m_hour = new Noun("time.hour");
 	private final static Noun m_day = new Noun("time.day");
-	final static Noun[] names = {m_tick, m_second, m_minute, m_hour, m_day};
-	final static long[] times = {50L, 1000L, 1000L * 60L, 1000L * 60L * 60L, 1000L * 60L * 60L * 24L};
+	private final static Noun m_week = new Noun("time.week");
+	private final static Noun m_month = new Noun("time.month");
```

Comment:
I'm not sure if we should support months (https://github.com/SkriptLang/Skript/issues/2220#issuecomment-509224048). It may seem counter intuitive to use `wait 1 month` on the 04-10 and have it end on 03-11. If you think we should add it anyway, you should add a note in timespan's description that a month is always considered 30 days.

Reply:
Other than no one probably will be able to wait 1 month or week at least :D I agree that it in such case it doesn't make sense but I guess we can just leave it like this with a default value of 30 days for months and 365 days for years and add a note about it as you suggested, opening the path for later contribution if anyone would like to support that.

I'm also working on adding `(day name|month name|..) of %date%`  and `%timespan%` (probably), there it could make more sense to support this